README.md

```diff
@@ -35,6 +35,30 @@ fi
 
 This ensures the version of `psycopg2` will match that of `psycopg2-binary`.
 
+
+## Running Tests
+
+To run initial setup
+
+* `hatch run setup`
+
+To run all unit tests
```

Comment:
Is there a formula for running one unit test or a file, or one integration test or a file/dir?